docs/blocks/columns.md

```diff
@@ -0,0 +1,17 @@
+## Columns
```

Comment:
Can you double check if we have different variants of the columns block we use? I think we have at least the case where it is making the image a background image and another one where we can reverse the way we break the columns.